testutils/clustermanager/gke.go

```diff
@@ -275,14 +196,18 @@ func (gc *GKECluster) Acquire() error {
 		return nil
 	}
 	// Perform GKE specific cluster creation logics
-	if gc.Request.ClusterName == "" {
+	if request.ClusterName == "" {
 		clusterName, err = getResourceName(ClusterResource)
 		if err != nil {
 			return fmt.Errorf("failed getting cluster name: '%v'", err)
 		}
+		request.ClusterName = clusterName
```

Comment:
what if we use `request.ClusterName` everywhere, and skip the var of `clusterName`?